etc/adminhtml/system.xml

```diff
@@ -48,8 +48,7 @@
                     <frontend_class>validate-number</frontend_class>
                 </field>
 
-                <field id="debug" translate="label comment" type="select" sortOrder="3" showInDefault="1"
-                       showInWebsite="1" showInStore="1">
+                <field id="debug" translate="label comment" type="select" sortOrder="3" showInDefault="1">
```

Comment:
so, you don't want a configuration of debug at the other levels than default one (but, in a multisite environment it could be interesting to log only at the website or store view, especially when the api_keys and merchant_id will be differ between this "levels"

Reply:
log with  multi store configuration need storeHelper dependency and generate circular dependency...

Reply:
keep it for a rainy day (or log factorisation)